modules/manage/pages/cluster-maintenance/compaction-settings.adoc

```diff
@@ -26,6 +26,56 @@ Compaction policy may be applied to a cluster or to an individual topic. If both
 * `compact,delete`: This combines both policies, deleting messages exceeding the retention period while compacting multiple versions of messages.
 
+== Tombstone record removal
+
+When a topic's messages should be deleted, for example as data is deleted from a source system, clients add a tombstone record to the message log. A tombstone contains the message key, and the value `null`. Tombstones signal to brokers and consumers that records with the same key prior to it in the log should be deleted. 
+
+You can specify how long Redpanda keeps these tombstones for compacted topics using both a cluster configuration property `tombstone_retention_ms` and a topic configuration property xref:reference:properties/topic-properties.adoc#deleteretentionms[`delete.retention.ms`]. If both are set, the topic-level tombstone retention policy overrides the cluster-level policy.
+
+[NOTE]
+====
+Redpanda does not currently remove tombstone records for compacted topics with Tiered Storage enabled. 
+
+You cannot enable `tombstone_retention_ms` if you have enabled any of the Tiered Storage cluster properties `cloud_storage_enabled`, `cloud_storage_enable_remote_read`, and `cloud_storage_enable_remote_write`.
+
+On the topic level, you cannot enable `delete.retention.ms` at the same time as the Tiered Storage topic configuration properties `redpanda.remote.read` and `redpanda.remote.write`.
+====
+
+To set the cluster-level tombstone retention policy, run the command:
+
+[,bash]
+----
+rpk cluster config set tombstone_retention_ms=100
+----
+
+You can unset the tombstone retention policy for a topic so it inherits the cluster-wide default policy:
+
+[,bash]
+----
+rpk topic alter-config <topic-name> --delete delete.retention.ms
+----
+
+To override the cluster-wide default for a specific topic:
+
+[,bash]
+----
+rpk topic alter-config <topic-name> --set delete.retention.ms=5
+----
+
+To disable tombstone removal for a specific topic:
+
+[,bash]
+----
+rpk topic alter-config <topic-name> --set delete.retention.ms=-1
+----
+
+Redpanda removes tombstones as follows:
+
+* For topics with a `compact,delete` cleanup policy: the standard garbage collection process, per the topic retention limit.
+* For topics with a `compact` only cleanup policy: The `delete.retention.ms` or `tombstone_retention_ms` value sets the time bound that a consumer has in order to see a complete view of the log with tombstones present before they are removed. 
+
+If obtaining a complete snapshot of the log, including tombstone records, is important to your consumers, set the tombstone retention value such that consumers have enough time for their reads to complete before tombstones are removed. If managing local disk space is a higher priority, consider setting the tombstone retention to a lower value, for example, less than 24 hours (86400000 ms).
```

Comment:
Taking inspiration from [kafka docs](https://docs.confluent.io/kafka/design/log_compaction.html#compaction-guarantees) once again, we could add a clarifying sentence similar to the following here:

> If obtaining a complete snapshot of the log, including tombstone records, is important to your consumers, set the tombstone retention value such that consumers have enough time for their reads to complete before tombstones are removed. Consumers may not see tombstone records if their reads take longer than `delete.retention.ms` to reach the head of the log.

Reply:
@WillemKauf is the last sentence in the paragraph still accurate and relevant? i.e. Can we say "If obtaining a complete snapshot of the log, including tombstone records, is important to your consumers... Consumers may not see tombstone records if their reads take longer than delete.retention.ms to reach the head of the log. _On the other hand, if managing local disk space is a higher priority_..."?

Reply:
I think something like the following might flow better?

> On the other hand, if more frequent cleanup of tombstone records is important for your workloads and space management, consider setting the tombstone retention to a lower value. For example, less than 24 hours (86400000 ms).

I don't have a great sentence yet, but a statement illustrating the trade-off between tombstone visibility for consumers and space management is still relevant and needed here.